extensions/cornerstone-3d/src/initMeasurementService.js

```diff
@@ -98,7 +98,7 @@ const connectToolsToMeasurementService = (
       // Todo: this should be changed when a measurement can include multiple annotations
       // in the future
       annotationAddedEventDetail.uid = annotationUID;
-      annotationToMeasurement(toolName, annotationAddedEventDetail);
+      addOrUpdate(toolName, annotationAddedEventDetail);
```

Comment:
@JamesAPetts I know we decided to do annotationToMeasurement and reverse, but in fixing unit tests, I think I came to a conclusion that it having `annotationToMeasurement` for updating a `measurement` sounds weird so just reverted back to what is in `v3-stable` now

Reply:
I don't know if I follow this, you are getting an annotation from the event, and using that to update the measurement. I think the naming of `annotationToMeasurement` was completely fine unless I've missed something?

Reply:
sure